docs/guides/anomaly-detection-configuration/ignore_small_changes.mdx

```diff
@@ -0,0 +1,62 @@
+---
+title: "ignore_small_changes"
+sidebarTitle: "ignore_small_changes"
+---
+
+```
+ignore_small_changes:
+  spike_failure_percent_threshold: [int]
+  drop_failure_percent_threshold: [int]
+```
+
+If defined, an anomaly test will fail only if all the following conditions hold:
+- The z-score of the metric within the detection period is anomoulous
+- The metric within the detection period is higher than `spike_failure_percent_threshold` percentages of the mean value in the training period, if defined.
+- The metric within the detection period is lower than `drop_failure_percent_threshold` percentages of the mean value in the training period, if defined
+
+Those settings can help to deal with 
```

Comment:
unfinished sentence?
Also I think it's a good idea to add a graph here, you can add one from Adam's Figma

Reply:
@ellakz 
Completed the sentence 😆 
Not sure there is a graph that reflects exactly those params, because with our last call it looks the same with just wider range